### ISSUE TITLE
Context contains invalid user._id value

### ISSUE THREAD

**Issue by vkiranmaniya** [feathersjs/feathers#3062]
### Steps to reproduce

Any basic feathers project with OAuth authentication enabled will work. After authentication, try the below code in any after `create` hook
```js
async (context) => {
    console.log(context.params.user._id)
}
```

### Expected behavior
As I'm using MongoDB with a mongoose service adapter, it should print an _id like `63eb6f4e3a56f35a62c048cf` 

### Actual behavior
its printing `new ObjectId("63eb6f4e3a56f35a62c048cf")`

### System configuration
**Module versions**:
```
    "@feathersjs/authentication": "^5.0.0-pre.37",
    "@feathersjs/authentication-client": "^5.0.0-pre.37",
    "@feathersjs/authentication-local": "^5.0.0-pre.37",
    "@feathersjs/authentication-oauth": "^5.0.0-pre.37",
    "@feathersjs/mongodb": "^5.0.0-pre.37",
    "@feathersjs/schema": "^5.0.0-pre.37",
    "@feathersjs/typebox": "^5.0.0-pre.37",
    "feathers-mongoose": "^8.5.1",
    "mongodb": "^5.0.1",
    "mongoose": "^6.9.1",
    "mongoose-schema-jsonschema": "^2.0.2"
```

**NodeJS version**:
v14.21.1

**Comment by vkiranmaniya** [comment]
Seems working when `context.params.user._id.toString()` is that standard behavior?

**Comment by daffl** [comment]
This has been addressed in 5.0.0-pre.38 via https://github.com/feathersjs/feathers/pull/3058 providing the `ObjectIdSchema` helper for Object ids. See https://feathersjs.com/api/databases/mongodb.html#objectidschema for the documentation.
